src/AES.h

```diff
@@ -1,9 +1,9 @@
 #ifndef _AES_H_
 #define _AES_H_
 
-#include <cstring>
-#include <iostream>
-#include <stdio.h>
+//#include <cstring>
+//#include <iostream>
```

Comment:
1. commenting these lines breaks the build
1. if some of them are not used, it will be better delete the lines, instead of comment

Reply:
Why resolved? I don't see any changes

Reply:
i forgotten click on upload. Sorry for that.